crates/daphne/src/vdaf/mod.rs

```diff
@@ -432,10 +432,7 @@ impl Encode for VdafAggregateShare {
 impl VdafConfig {
     pub(crate) fn uninitialized_verify_key(&self) -> VdafVerifyKey {
         match self {
-            Self::Prio3Draft09(Prio3Config::SumVecField64MultiproofHmacSha256Aes128 { .. })
-            | Self::Prio2 { .. }
-            | Self::Prio3(..) => VdafVerifyKey::L32([0; 32]),
-            Self::Prio3Draft09(..) => VdafVerifyKey::L16([0; 16]),
```

Comment:
Since we dropped the old variants of Prio3, the seed size is 32 bytes for every Prio3 variant we support :)